rest/backend/src/main/scala/io/udash/rest/server/ExposesREST.scala

```diff
@@ -34,7 +34,10 @@ abstract class ExposesREST[ServerRPCType : UdashRESTFramework#ValidServerREST](l
 
   def handleRestCall(req: HttpServletRequest, httpMethod: Class[_ <: RESTMethod])(implicit ec: ExecutionContext): Future[String] = {
     val invocations = List.newBuilder[RawInvocation]
-    val path: Array[String] = Option(req.getPathInfo).map(_.stripPrefix("/").split("/")).getOrElse(Array.empty[String])
+    val path: Array[String] =
+      Option(req.getRequestURI.stripPrefix(req.getServletPath))
```

Comment:
Can `.stripPrefix` return null? I'd be more worried about `getRequestURI`. Also use `Opt` :rocket: